### DISCUSSION TITLE
Security

### DISCUSSION THREAD

**Discussion by shamsullah-shams** [community/community#111259] [Copilot Conversations]
I don't want to give fork/clone/download rights

**Comment by gil-son** [comment]
You can change the visualization of your repository to private

**Comment by ghostinhershell** [comment]
Hi @shamsullah-shams,

Thanks so much for reaching out in our community. You can allow or prevent the forking of a specific private repository owned by an organization: [Managing the forking policy for your repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-forking-policy-for-your-repository).
